scss/components/_button.scss

```diff
@@ -161,10 +161,9 @@ $button-transition: background-color 0.25s ease-out, color 0.25s ease-out !defau
 @mixin button-disabled($color: $primary-color) {
   opacity: $button-opacity-disabled;
   cursor: not-allowed;
-
+  background-color: $color;
```

Comment:
This does not change anything. If fact, like explained at https://github.com/zurb/foundation-sites/issues/9617#issuecomment-271232428, it makes the background of `[disabled]` be overridden by the one of `:focus` and `:hover` if they were defined after.

But maybe it needs more discussions to know if it is the expected behavior. We could consider `[disabled]` as a "low-level modifier" (so should have a specificity of `0.0.1` and should be overridable), or as a "low-level state" (so should have a specificity of `0.0.2` and should not be overridable).